src/threading.ts

```diff
@@ -1173,7 +1173,7 @@ export class JVMThread implements Thread {
     } else {
```

Comment:
I let the `util.initString` use the bootstrap classloader cause my hunch was that it needn't / shouldn't use any other.